core/os/os.cpp

```diff
@@ -95,7 +95,6 @@ uint64_t OS::get_system_time_msecs() const {
 }
 
 void OS::debug_break(){
-
```

Comment:
`clang-format` does this weird thing on method definitions which end with an unnecessary semicolon, it's pretty messed up. It would be good to fix all occurrences of this (in a different PR):
```
$ rg "\(\)\{" -g'!thirdparty'
servers/audio/audio_driver_dummy.cpp
134:AudioDriverDummy::~AudioDriverDummy(){

core/command_queue_mt.h
287:            virtual void post(){};
288:            virtual ~CommandBase(){};

core/os/os.h
521:    virtual void force_process_input(){};

core/os/thread_dummy.h
54:     virtual void lock(){};
55:     virtual void unlock(){};

core/os/os.cpp
96:void OS::debug_break(){
520:void OS::native_video_pause(){
524:void OS::native_video_unpause(){
528:void OS::native_video_stop(){

core/ring_buffer.h
221:    ~RingBuffer<T>(){};

servers/physics/physics_server_sw.cpp
1445:void PhysicsServerSW::sync(){
1586:PhysicsServerSW::~PhysicsServerSW(){

servers/physics_2d/physics_2d_server_sw.cpp
1461:Physics2DServerSW::~Physics2DServerSW(){

core/io/file_access_buffered_fa.h
154:    FileAccessBufferedFA(){

servers/physics/joints/jacobian_entry_sw.h
57:     JacobianEntrySW(){};

drivers/coreaudio/audio_driver_coreaudio.cpp
704:AudioDriverCoreAudio::~AudioDriverCoreAudio(){};

core/vmap.h
205:    _FORCE_INLINE_ VMap(){};

servers/arvr/arvr_interface.cpp
125:ARVRInterface::~ARVRInterface(){};

servers/arvr/arvr_positional_tracker.cpp
235:ARVRPositionalTracker::~ARVRPositionalTracker(){

core/object.h
581:    _FORCE_INLINE_ static void register_custom_data_to_otdb(){};

drivers/dummy/rasterizer_dummy.h
754:    RasterizerStorageDummy(){};
764:    void canvas_begin(){};
765:    void canvas_end(){};

platform/windows/key_mapping_windows.h
42:     KeyMappingWindows(){};

platform/x11/key_mapping_x11.h
44:     KeyMappingX11(){};

platform/uwp/thread_uwp.cpp
67:ThreadUWP::ThreadUWP(){
71:ThreadUWP::~ThreadUWP(){

platform/iphone/ios.mm
84:iOS::iOS(){};

platform/iphone/in_app_store.mm
337:InAppStore::~InAppStore(){};

platform/iphone/icloud.mm
357:ICloud::~ICloud(){};

platform/iphone/game_center.mm
398:GameCenter::~GameCenter(){};

platform/haiku/key_mapping_haiku.h
35:     KeyMappingHaiku(){};

modules/gdscript/gdscript_tokenizer.h
180:    virtual ~GDScriptTokenizer(){};

editor/rename_dialog.h
51:     void _cancel_pressed(){};
115:    ~RenameDialog(){};

modules/camera/camera_win.cpp
56:CameraFeedWindows::CameraFeedWindows(){
78:void CameraFeedWindows::deactivate_feed(){
85:void CameraWindows::add_active_cameras(){
96:CameraWindows::~CameraWindows(){

scene/main/viewport.cpp
177:    TooltipPanel(){};
185:    TooltipLabel(){};

scene/3d/vehicle_body.cpp
47:     btVehicleJacobianEntry(){};

scene/3d/arvr_nodes.cpp
170:ARVRCamera::ARVRCamera(){
174:ARVRCamera::~ARVRCamera(){
389:ARVRController::~ARVRController(){
528:ARVRAnchor::~ARVRAnchor(){
619:ARVROrigin::~ARVROrigin(){

scene/3d/proximity_group.cpp
212:ProximityGroup::~ProximityGroup(){
```

Reply:
By the way, should we also add the missing space between `()` and `{`?

Reply:
(The above was from the `3.2` branch. I'm now working on fixing them.)

Reply:
@Calinou That's what I was commenting about yes :)